src/style.css

```diff
@@ -1 +1,81 @@
 /* add styles here */
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300px;
+  line-height: 21.86px;
+  text-align: left;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(/src/fonts/Avenir-Book.ttf) format(ttf);
+  src: url(/src/fonts/Avenir-Heavy.ttf) format(ttf);
+  src: url(/src/fonts/Avenir.ttc) format(ttc);
+  font-weight: normal;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+  font-family: Avenir, sans-serif;
+  padding: 0;
+  width: 100vw; /* Полная ширина окна */
+  height: 100vh; /* Полная высота окна */
```

Comment:
Setting `height` to `100vh` on the `body` tag is not recommended as it can lead to unwanted overflow issues with the content. Use min-height if you want to ensure the body takes at least the viewport height.